packit.yaml

```diff
@@ -13,6 +13,9 @@ srpm_build_deps:
   - glib2-devel
   - make
   - systemd-devel
+files_to_sync:
+  - packit.yaml
+  - cockpit.spec
```

Comment:
Note: This will sync everything *but* the `%changelog`, see [docs](https://packit.dev/docs/configuration/#sync_changelog). This is what we want, and what our own release machinery does as well. The changelog just gets a new entry based on the upstream release tag.

Reply:
I actually don't understand this part at all.  Let's talk.

Reply:
Not necessary any more, dropped.